base/security_unittest.cc

```diff
@@ -54,8 +54,7 @@ void OverflowTestsSoftExpectTrue(bool overflow_detected) {
     BUILDFLAG(IS_APPLE)
     // Sadly, on Linux, Android, and OSX we don't have a good story yet. Don't
     // fail the test, but report.
-    printf("Platform has overflow: %s\n",
-           !overflow_detected ? "yes." : "no.");
+    printf("Platform has overflow: %s\n", !overflow_detected ? "yes." : "no.");
```

Comment:
Is this change necessary? If no, maybe we should keep it the same as upstream?

Reply:
This was accidental, reverted